_data/Damian96/Damian96.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Damian
+institution: City College, International Faculty of the University of Sheffield
+profile_pic: Damian96.png
+quote: eval "php artisan inspire"
```

Comment:
Remove double quotes from begin and end.

Reply:
Yes, sure!